crates/aptos-rest-client/src/lib.rs

```diff
@@ -620,6 +620,9 @@ impl Client {
                         )
                     );
                 },
+                Err(err) => {
+                    debug!("Fetching error, will retry: {}", err);
```

Comment:
nit: can we maybe also print the default delay to output how long we'll wait until we retry?

Reply:
This and a few other messages will need to be standardized and cleaned up, but this is fine.